modules/samples/producers/kafka-performance/src/main/java/org/wso2/carbon/sample/kafka/performance/KafkaClient.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.sample.kafka.performance;
+
+import com.google.gson.JsonObject;
+import kafka.javaapi.producer.Producer;
+import kafka.producer.KeyedMessage;
+import kafka.producer.ProducerConfig;
+import org.apache.log4j.Logger;
+
+import java.text.DecimalFormat;
+import java.util.Properties;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
+
+
+public class KafkaClient {
+    private static Logger log = Logger.getLogger(KafkaClient.class);
+
+    private static String url;
+    private static String topic;
+    private static long noOfEvents;
+    private static int elapsedCount;
+    private static int noOfPublishers;
+
+    //Atomic Variables
+    private AtomicLong count;
+    private AtomicLong lastIndex;
+    private AtomicBoolean calcInProgress;
+
+
+    public KafkaClient() {
+        count = new AtomicLong(0);
+        lastIndex = new AtomicLong(0);
+        calcInProgress = new AtomicBoolean(false);
+    }
+
+    public static void main(String args[]) {
```

Comment:
@sajithshn 
Thank you for the suggestion. We have handled this in performance client build.xml which echoes the command line arguments to be passed. Please refer : https://github.com/thiliA/product-cep/blob/63b382c48fd42a6ab56b0bc04664e8d46a87cc2c/modules/samples/producers/kafka-performance/build.xml (line 70)